### PR TITLE
Paulina/3 post validation

### PR DESCRIPTION
**Pull request by PaulinaLL**

### CODE REVIEW

File: validation/validation.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
+const Joi = require("@hapi/joi");
```
Comment (oliverwebr) [inline validation/validation.js]: Lets put the validation.js into our middleware folder

File: validation/validation.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    class: Joi.string().min(3),
+
+    location: Joi.any().allow("BER", "HH", "DUS")
+  });
+
+  const value = schema.validate(req.body);
+  if (!value.error) {
+    next();
+  } else {
+    res.send(value.error.message);
```
Comment (oliverwebr) [inline validation/validation.js]: At the moment we send back the response with a status code `200` which communicates "Ok", we should use the status code `400` ["Bad Request"](https://en.wikipedia.org/wiki/List_of_HTTP_status_codes#400)

File: validation/validation.js (hunk 1 above)
Comment (PaulinaLL) [reply validation/validation.js]: done

File: validation/validation.js (hunk 2 above)
Comment (PaulinaLL) [reply validation/validation.js]: done, hopefully..

**Pull request merged**